tests/test_exceptions.py

```diff
@@ -1,35 +1,91 @@
 import pytest
 
-from wxflow import WorkflowException
+from wxflow import WorkflowException, WorkflowKeyError, WorkflowTypeError
 
 # ----
 
 
-class TestError(WorkflowException):
+class TestWorkflowException(WorkflowException):
     """
     Description
     -----------
 
-    This is the base-class for exceptions encountered within the
-    wxflow/errors unit-tests module; it is a sub-class of Error.
+    This is the base-class for generic exceptions encountered within the
+    wxflow/errors unit-tests module; it is a sub-class of WorkflowException.
+
+    """
+
+
+class TestWorkflowKeyError(WorkflowKeyError):
+    """
+    Description
+    -----------
+
+    This is the base-class for KeyError exceptions encountered within the
+    wxflow/errors unit-tests module; it is a sub-class of WorkflowKeyError.
+
+    """
+
+
+class TestWorkflowTypeError(WorkflowTypeError):
+    """
+    Description
+    -----------
+
+    This is the base-class for TypeError exceptions encountered within the
+    wxflow/errors unit-tests module; it is a sub-class of WorkflowTypeError.
 
     """
 
 # ----
 
 
-def test_errors() -> None:
+def test_workflow_exception() -> None:
+    """
+    Description
+    -----------
+
+    This function provides a unit test for the WorkflowException class.
+
+    """
+
+    # Raise the base-class exception.
+    with pytest.raises(TestWorkflowException):
+        msg = "Testing WorkflowException raise."
+        raise TestWorkflowException(msg=msg)
```

Comment:
Is it necessary to have a `TestWorkflowException` class?  Could this instead simply raise a `WorkflowException`?

```suggestion
    with pytest.raises(WorkflowException):
        msg = "Testing WorkflowException raise."
        raise WorkflowException(msg)
```

Reply:
@DavidHuber-NOAA Not sure. I was following the example already present in this script. I'm also having trouble testing locally on hera because pytest isn't available. Let me try it and push and see what happens.

Reply:
@DavidNew-NOAA Just an FYI: you should be able to get it by setting up a virtual environment:

```bash
. $HOMEgfs/dev/ush/gw_setup.sh  # Easy way to get a new enough Python version
cd $wxflow
python -m venv venv
. venv/bin/activate
pip install .[dev]
pytest -v
```

Reply:
@DavidHuber-NOAA Thanks! I will add that to my mental toolbox